pkg/polylog/polyzero/logger_example_test.go

```diff
@@ -0,0 +1,37 @@
+package polyzero_test
+
+import (
+	"os"
+
+	"github.com/rs/zerolog"
+
+	"github.com/pokt-network/poktroll/pkg/polylog/polyzero"
+)
+
+func ExampleNewLogger() {
+	// Use whichever zerolog level you need.
+	level := zerolog.InfoLevel
+	// Specify the lowest level to log. I.e.: calls to level methods "lower"
+	// than this will be ignored.
+	levelOpt := polyzero.WithLevel(level)
+
+	// Construct logger.
+	// NB: adding WithOutput is optional; defaults to os.Stderr. It is needed
+	// here to print to stdout for testable example purposes.
+	logger := polyzero.NewLogger(levelOpt, polyzero.WithOutput(os.Stdout))
+
+	// All level methods are always available, but will only log if the level
```

Comment:
From this comment https://github.com/pokt-network/poktroll/pull/211/files#r1403754560 it seems like the levels have a heirarchy: Debug -> Info -> Err -> Warn maybe idk, if so PUC and reference this somewhere or correct my understanding